src/plugins/presentation_panel/public/panel_component/_presentation_panel.scss

```diff
@@ -106,12 +105,12 @@
   */
 
 .embPanel__optionsMenuButton {
-  background-color: transparentize($euiColorDarkestShade, .9);
+  background-color: transparentize($euiColorLightShade, .5);
   border-bottom-right-radius: 0;
   border-top-left-radius: 0;
 
   &:focus {
-    background-color: $euiFocusBackgroundColor;
+    background-color: transparentize($euiColorLightestShade, .5);
```

Comment:
There are four scenarios to consider for the colours of the `optionsMenuButton`:

1. Kibana is in light mode + the background of the image is light
    
    | Before | After |
    |--------|--------|
    | ![image](https://github.com/elastic/kibana/assets/8698078/37ebe186-7351-4402-973d-ad4151ac3a9c) | ![image](https://github.com/elastic/kibana/assets/8698078/7153d77e-90ae-42a3-9b7f-aa3cc52c490e) | 

2. Kibana is in light mode + the background of the image is dark
    
    | Before | After |
    |--------|--------|
    | ![image](https://github.com/elastic/kibana/assets/8698078/dfd41b62-5de0-4720-8d34-e4ac4dd83ecd) | ![image](https://github.com/elastic/kibana/assets/8698078/750a5486-12b2-43b4-82af-f8edec998f06) | 

3. Kibana is in dark mode + the background of the image is light
    
    | Before | After |
    |--------|--------|
    | ![image](https://github.com/elastic/kibana/assets/8698078/45d90805-058f-4551-a21c-0e8590b58995) | ![image](https://github.com/elastic/kibana/assets/8698078/38d18c74-3683-4174-9839-380e8f622db2) | 

4. Kibana is in dark mode + the background of the image is dark
    
    | Before | After |
    |--------|--------|
    | ![image](https://github.com/elastic/kibana/assets/8698078/2096e097-4b74-4e1a-9b42-ac35bd676c93) | ![image](https://github.com/elastic/kibana/assets/8698078/b7444d6d-c088-41b4-9a2b-4c58b4c75f71) | 


Notice in the above screenshots that this styling change fixes scenarios 2 and 4 (i.e. it makes it so that `optionsMenuButton` is visible with good contrast regardless of the theme + image background combo) while keeping appearances more-or-less the **same** in scenarios 1 and 3.